public/data/vasantisuthar.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "VASANTI SUTHAR",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/vasantisuthar.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/vasantisuthar",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vasanti_suthar",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Showwcase",
+      "url": "https://www.showwcase.com/vasanti-suthar",
+      "icon": "showwcase"
```

Comment:
Please use `link`, `globe`, or any other available icons here.

Reply:
the "showwcase" will by default show globe icon as it not available

Reply:
Yes, you can add showwcase. @Panquesito7 there is a default icon that it will fall back to until showwcase is added so this won’t cause any problems and it will automatically work once the icon is added.

Reply:
> the "showwcase" will by default show globe icon as it not available

yup.